aries_vcx/Cargo.toml

```diff
@@ -31,11 +33,16 @@ serde = "1.0.97"
 serde_json = "1.0.40"
 serde_derive = "1.0.97"
 regex = "1.1.0"
+reqwest = "0.11.10"
 rust-base58 = "0.0.4"
 base64 = "0.10"
 openssl = { version = "0.10.35" }
 num-traits = "0.2.0"
 libvdrtools = { git = "https://gitlab.com/left-arm/vdr-tools.git", rev = "debf776c8" }
+# indy alternatives ----
+indy-vdr = "0.3.4"
+indy-credx = { git = "https://github.com/anonyome/indy-shared-rs.git", rev = "7342bc624d23ece8845d1a701cd2cdc9cd401bb0" }
```

Comment:
What kind of changes has been done in your fork? Do you plan to open PR?

Reply:
i've added a lengthy comment in the toml justifying this fork, as well as instructions on how to monitor it / resolve in the future. it's unfortunate that we have to use a fork, but let me know if you can think of a better temporary solution